app/connectors/SubscriptionConnector.scala

```diff
@@ -42,35 +38,19 @@ class SubscriptionConnector @Inject() (
 
   private val logger = Logger(this.getClass)
 
-  def get(
-    pptReferenceNumber: String
-  )(implicit hc: HeaderCarrier): Future[Either[EisFailure, SubscriptionDisplayResponse]] = {
+  def get(pptReferenceNumber: String)(implicit hc: HeaderCarrier): Future[Either[EisFailure, SubscriptionDisplayResponse]] = {
+
     val timer = metrics.defaultRegistry.timer("ppt.subscription.get.timer").time()
     httpClient.GET[HttpResponse](appConfig.pptSubscriptionUrl(pptReferenceNumber))
       .map {
         response =>
-          if (Status.isSuccessful(response.status))
-            Try(response.json.as[SubscriptionDisplayResponse]) match {
-              case Success(successfulGetResponse) =>
-                logger.info(
-                  s"Retrieved subscription for ppt reference number [$pptReferenceNumber]"
-                )
-                Right(successfulGetResponse)
-              case _ =>
-                throw UpstreamErrorResponse.apply(
-                  "Failed to parse successful get subscription response",
-                  Status.INTERNAL_SERVER_ERROR
-                )
-            }
-          else
-            Try(response.json.as[EisFailure]) match {
-              case Success(failedGetResponse) => Left(failedGetResponse)
-              case _ =>
-                throw UpstreamErrorResponse.apply(
-                  "Failed to parse failed get subscription response",
-                  Status.INTERNAL_SERVER_ERROR
-                )
-            }
+          if (Status.isSuccessful(response.status)) {
```

Comment:
know this hasnt changed, but is this 200 or can it be any 2XX? feels like it could go wrong for 204